app/src/main/java/org/wikipedia/feed/image/FeaturedImageCardView.kt

```diff
@@ -26,9 +28,16 @@ class FeaturedImageCardView(context: Context) : DefaultFeedCardView<FeaturedImag
                 image(it.baseImage())
                 description(it.description())
                 header(it)
+                updateLabelsToWikiLang()
                 setClickListeners()
             }
         }
+
+    private fun updateLabelsToWikiLang() {
+        binding.viewFeaturedImageCardDownloadButton.text = L10nUtil.getStringForArticleLanguage("en", R.string.view_featured_image_card_download)
```

Comment:
Sorry, on second thought, the Picture of the Day card should probably appear in the device's language, since Commons is language-agnostic.
It definitely makes no sense to hard-code this to English.

Reply:
Makes sense. I was meaning to use the app lang instead of the hardcoded "en", but I feel even that will not make sense. Removed this.